lib/iris/fileformats/netcdf.py

```diff
@@ -1494,9 +1494,18 @@ def _create_cf_cell_measure_variable(self, cube, dimension_names,
         cf_dimensions = [dimension_names[dim] for dim in
                          cube.cell_measure_dims(cell_measure)]
 
+        # Get the data values
+        data = cell_measure.data
+
+        # Disallow saving of *masked* cell measures.
+        if ma.is_masked(data):
+            # We can't really save properly if there are masked points, because
+            # we don't (yet) record an appropriate fill_value property.
+            msg = "Currently cannot save cell measures with missing data."
```

Comment:
I'd drop the "currently" -- we _just don't support this_, and that's just how it is unless someone asks for it.